api/src/main/java/org/pmiops/workbench/actionaudit/auditors/EgressEventAuditorImpl.kt

```diff
@@ -154,7 +154,7 @@ constructor(
                 actionId = actionIdProvider.get(),
                 actionType = ActionType.DETECT_HIGH_EGRESS_EVENT,
                 agentType = AgentType.SYSTEM,
-                agentId = 0,
+                agentIdMaybe = 0,
```

Comment:
you should be able to leave this null now. Looks like the local convention is to explicitly set the optionals for clarity. Otherwise I'd generally say to leave out the args that have defaults.

Reply:
done